pypeit/core/parse.py

```diff
@@ -680,19 +679,11 @@ def sec2slice(subarray, one_indexed=False, include_end=False, require_dim=None,
         require_dim (:obj:`int`, optional):
             Test if the string indicates the slice along the proper
             number of dimensions.
-        transpose (:obj:`bool`, optional):
-            Transpose the order of the returned slices.  The
-            following are equivalent::
-                
-                tslices = parse.sec2slice('[:10,10:]')[::-1]
-                tslices = parse.sec2slice('[:10,10:]', transpose=True)
-
-        binning_raw (:obj:`str`, optional):
-            The image binning_raw.  The `subarray` string is always expected to be for an *unbinned* image.
-            This binning_raw keyword is used to adjust the slice for an image that is binned.
-            The string must be a comma-separated list of number providing the binning_raw along the relevant axis.
-            Note that since sec2slice works in the native fits coordinate system/convention that is written to the
-            image headers, this binning_raw is *FLIPPED* relative to PypeIt's binning conventions of specXspat
+        binning (:obj:`str`, optional):
+            Assume the slice is for an unbinned array and adjust the
+            returned slice for this binning in each dimension.  If two
+            dimensional, the format of this string must be, e.g., `1,2`
+            for unbinned rows and a factor of 2 binning along columns.
```

Comment:
Did you fix all the transposing stuff. Not sure I follow how this all got removed.

Reply:
Yeah.  I've made it so that all PypeIt functions return spectral then spatial image dimensions to match the convention.  The main change is that `Spectrograph.get_image_section` now always returns the arrays ordered spectral then spatial.  Also see the docstring for `load_raw_frame`.

Reply:
Great!